migration/20200127-remove-work-random.sql

```diff
@@ -3,4 +3,4 @@ drop index if exists ix_works_audience_target_age_quality_random;
 drop index if exists ix_works_audience_fiction_quality_random;
 
 -- Remove the works.random column itself.
-ALTER TABLE works DROP COLUMN if exists random;
+ALTER TABLE works DROP COLUMN if exists random CASCADE;
```

Comment:
This is a leftover change from my QA of https://jira.nypl.org/browse/SIMPLY-2518. Our QA server had an old materialized view lying around for some reason. This change makes the column drop also drop the materialized view, improving the robustness of the migration script.